### DISCUSSION TITLE
NSclient++ ssl options

### DISCUSSION THREAD

**Discussion by louwar** [mickem/nscp#815] [Q&A]
Hi all,
I was able to avoid usage of SSLv2 SSLv3 and TLSv1, from Nagios agent in Windows environment, 
setting this row 
ssl options = no-sslv2,no-sslv3,no-tlsv1
in the nsclient.ini file

How can I avoid also to use TLSv1.1 ?
I tried with no-tlsv1.1 or no-tlsv1_1 but doesn't work
I have the 0.5.2.41 version of the software.

In Linux environment, I put this row
ssl_version=TLSv1.2+
in the /etc/nagios/nrpe.cfg file and I use just TLS1.2+

I want to obtain the same in Windows

Can anyone help me?
Thanks

**Comment by mickem** [comment]
I just added flags to configure this which will be in nightly builds in the next few days...

**Comment by rsudre** [comment]
nsclient++ web interface accepts the password, but does not leave the login screen. Could anyone help me how to solve it?

nsclient version nscp-00.5.41-x64 installed on windows 2008server.

Thanks!

**Comment by mickem** [comment]
Latest version has a new Web Interface so please reopen this if you still have issues
